content/en/docs/tasks/security/auth-sds/index.md

```diff
@@ -354,8 +354,7 @@ To enable the pod security policy, perform the following steps:
 
 Currently, the SDS identity provision flow has the following caveats:
 
-* You still need secret volume mount for enabling the control plane security.
-  Enabling SDS for the control plane security remains a work in progress.
+* SDS for the control plane security is enabled by default, and it is alpha version.
```

Comment:
```suggestion
* The SDS feature is currently in Alpha phase.
```

Reply:
I don't think we need to mention too much about the control plane security. Our users should naturally think the control plane security is using SDS.